tools/jdbc/src/test/java/org/duckdb/test/TestDuckDBJDBC.java

```diff
@@ -495,6 +495,48 @@ public static void test_throw_wrong_datatype() throws Exception {
 		conn.close();
 	}
 
+	public static void test_list_metadata() throws Exception {
+		Connection conn = DriverManager.getConnection("jdbc:duckdb:");
+		Statement stmt = conn.createStatement();
```

Comment:
Can you please add the missing close calls or wrap these in try-with-resources?

Reply:
👍🏻

Reply:
Done thanks!